run/events-gcs/main.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [START run_events_pubsub_server_setup]
+
+# [START run_events_pubsub_server_setup]
+import os
+
+from flask import Flask, request
+
+
+app = Flask(__name__)
+# [END run_events_pubsub_server_setup]
+
+
+# [START run_events_pubsub_handler]
+@app.route('/', methods=['POST'])
+def index():
+    envelope = request.get_json()
+    print(f'Storage Headers: {request.headers}\n Storage Data: {envelope}', 200)
```

Comment:
Are we not doing any checks to see is headers or envelope exist? Do we not want to do any more unmarshalling of the data?

Reply:
We aren't doing any more checks as the product should deliver correct payloads.

Let's log this out:
`GCS CloudEvent type: ${ce-subject}`

We can keep the:
`print(f'Storage Headers: {request.headers}\n Storage Data: {envelope}'`
for now if we want since that's pretty useful information.

See:
go/events-for-cloud-run-devrel-code